### PR TITLE
solve responsite problem for images

### PR DESCRIPTION
**Pull request by moheb2000**
When we add an image in posts in blogdown like R plots, on smartphones plot is really big and don't see in the screen.

### CODE REVIEW

File: static/css/main.css [hunk 1]
```diff
@@ -29,6 +29,11 @@ code {
   background-color: rgba(0, 0, 0, 0.02);
 }
 
+img {
+  max-width: 100%;
+  height: auto;
```
Comment (yihui) [inline static/css/main.css]: Is it necessary to define `height`?

**Review by yihui — APPROVED** [review]
Sounds good. Thanks!

File: static/css/main.css (hunk 1 above)
Comment (moheb2000) [reply static/css/main.css]: Actually it's not, but these lines are from original hugo theme and I think it's better to not changing them.
but yes it is not necessary.

Comment (yihui) [inline static/css/main.css]:
```suggestion
```

**Review by yihui — APPROVED** [review]
Okay. I'll delete the height. Thanks!

**Pull request merged**